app/tag/[category]/loading.tsx

```diff
@@ -0,0 +1,21 @@
+function loading() {
```

Comment:
The reason I created multiple loading components is each loading component works for different pages. I could make one loading component and the loading UI could be just a "rotating thing", but I make different loading component so that they can show different skeleton based on in what page is this. And according to next js docs multiple loading components for each page is totally okay )